core/domain/collection_services.py

```diff
@@ -1206,3 +1213,23 @@ def index_collections_given_ids(collection_ids):
     search_services.index_collection_summaries([
         collection_summary for collection_summary in collection_summaries
         if collection_summary is not None])
+
+
+def get_acquired_skills_of_user_given_collection_id(user_id, collection_id):
+    """Returns the acquired skills of the user identified by user_id
+    for a given collection.
+
+    Args:
+        user_id: str. The id of the user.
+        collection_id: str. The id of the collection.
+
+    Returns:
+        list. A list of skill ids acquired by the user.
```

Comment:
list(str). (Make sure it's really a list -- see above.)

Reply:
Updated.